src/api/routers/participantsRouter.ts

```diff
@@ -90,6 +90,7 @@ export function createParticipantsRouter() {
 
   participantsRouter.post('/', async (req, res) => {
     try {
+      const traceId = req?.headers?.traceId?.toString() ?? '';
```

Comment:
This is repeated quite a few times. It might be worth creating a helper function, maybe in `loggingHelpers.ts`?

Reply:
done! created `getTraceId` in `loggingHelpers.ts`